main.cpp

```diff
@@ -50,86 +50,105 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
+
+
+/* My exampes */
+
+/*apologies some were a bit tongue-in-cheek as I was running out of things to think of in the context of a working day - I don't really have a 5PM alarm! */
+
+
+//  1)
+//  Noun:       spindle
+//  action 1:   spindle moves in X
+spindle.moveX();
+//  action 2:   spindle checks on/off status
+spindle.isSpinning(); 
```

Comment:
The action word you used is `check`
your psuedo-code doesn't use this word. 
what you want to write here is `spindle.checkOnOffStatus();` 
- [ ] revise